waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/DatabaseMappingImpl.java

```diff
@@ -483,6 +484,12 @@ public GetTablesResult transformOutboundGetTablesResult(GetTablesResult result)
     return result;
   }
 
+  @Override
+  public PartitionValuesRequest transformInboundPartitionValuesRequest(PartitionValuesRequest req) {
+    req.setDbName(transformInboundDatabaseName(req.getDbName()));
```

Comment:
Shouldn't the value inside `setDbName` be `metaStoreMapping.transformInboundDatabaseName(....)' (based on other similar methods in this class)?

Reply:
Hmmm, now that I look at it, the internal method `transformInboundDatabaseName()` also delegates to `metastoreMapping.transformInboundDatabaseName()`. So it doesn't actually make a difference but it's a bit confusing reading the code. @patduin do you think it would be worth updating all the usages to just call `transformInboundDatabaseName()` everywhere so it's consistent? Or not worth the bother?

Reply:
I'm not really bothered